### ISSUE TITLE
Paste and match style

### ISSUE THREAD

**Issue by davekuhar** [sivaramsi/manageyum#10]
"Paste and match style" is a standard feature in most Mac apps. When I copy text to be pasted into an email (usually from a website) I expect to be able to paste it into an email without carrying the formatting along with it. I don't need bold 24-pt. red text pasted into the body of an email that's composed with 12-pt. black text, so I use Paste and Match Style (command-shift-V in Chrome) to dump the text into the message cleanly.

Manageyum lacks this feature, so I'm forever pasting first into just about any other program to strip the formatting, copying that cleaned up text to the clipboard, then pasting into Manageyum. Anything you can do about this?

**Comment by weavermedia** [comment]
As a workaround you can use CMD+\ in Gmail to remove all formatting. Slightly less painful:

- paste formatted text into regular Gmail compose window
- select the text you just pasted
- hit CMD+\ to remove all formatting and reset to regular Gmail email style

**Comment by davekuhar** [comment]
That's basically what I'm doing now, though it's funny that your suggestion includes pasting into the regular Gmail compose window. I'm using Manageyum specifically to avoid having a few Gmail tabs open all the time in Chrome. ;)

**Comment by weavermedia** [comment]
Indeed. I'm using Manageyum to keep track of 6-7 different Gmail accounts so I don't have to have them all open in Chrome. Very useful for that. Even more useful when Profiles become available.

**Comment by davekuhar** [comment]
It's handy for the tabs I normally have to keep open for the several SaaS's I use daily. Looking forward to seeing how Manageyum evolves.
